resources/views/app/includes/contents/episode.blade.php

```diff
@@ -1,50 +1,42 @@
 <div class="row">
```

Comment:
**Code Quality - Nitpick**
@andela-tolotin Please make use of the required font-family, download if need be.